lib/components/Label.js

```diff
@@ -23,9 +33,11 @@ const Label = ({
       )}
     >
       <div className="flex flex-row">
-        <label className="m-0">
+        <label className="m-0" {...otherProps}>
           {children}
-          {required && "*"}
+          {required && (
+            <i className="ri-asterisk ri-xs align-text-bottom ml-0.5"></i>
```

Comment:
@yedhink _a After you suggested if l also feel the icon is an overkill. Also the icon doesn't look as good as `*`.

Reply:
@edwinbbu Should we wait for @goutham-subramanyam's input or can we make the change and merge once done?

Reply:
@yedhink can you give ``<span className="ml-0.5">*</span>`` and see how it looks.

Reply:
@edwinbbu 
Looks fine to me:
![image](https://user-images.githubusercontent.com/31244999/101136182-f31d9900-3632-11eb-978c-5dfb1192016a.png)

Shall i persist this change?

Reply:
Sure.

Reply:
Looks good 👍